optuna/structs.py

```diff
@@ -84,11 +85,9 @@ class FrozenTrial(
             set by Optuna.
         intermediate_values:
             Intermediate objective values set with :func:`optuna.trial.Trial.report`.
-        params_in_internal_repr:
```

Comment:
Let's keep `params_in_internal_repr` and `trial_id` in the document and clarify that they are for Optuna internal use.

Reply:
Thanks. I've fixed it.